tracer/src/Datadog.Tracer.Native/debugger_rejit_preprocessor.cpp

```diff
@@ -133,9 +133,9 @@ ULONG DebuggerRejitPreprocessor::PreprocessLineProbes(const std::vector<ModuleID
     return rejitCount;
 }
 
-void DebuggerRejitPreprocessor::EnqueuePreprocessLineProbes(const std::vector<ModuleID>& modulesVector,
-    const LineProbeDefinitions& lineProbes,
-    std::promise<std::vector<MethodIdentifier>>* promise) const
+void DebuggerRejitPreprocessor::EnqueuePreprocessLineProbes(const std::vector<ModuleID> modulesVector,
+                                                            const std::vector<LineProbeDefinition> lineProbes,
```

Comment:
is it expected to have a copy of the `lineProbes`. wouldn't it had a performance issue ?

Reply:
Sharp eyes! I accidently commented it out during a templating change that annoyed me due to linkage issues so I tried to rule out different parts and left this one as it was.
I will revive this line. Thanks!

Reply:
Done in 1bed3f68dfca764bd20a60fea21287b9274e0258.

Reply:
Mistakenly sent this message in the incorrect comment. I was referring to: https://github.com/DataDog/dd-trace-dotnet/pull/3955#discussion_r1149667825